corehq/ex-submodules/couchexport/models.py

```diff
@@ -969,7 +970,11 @@ def get_attachment_name(self):
         return hashlib.md5(six.text_type(self.configuration.filename).encode('utf-8')).hexdigest()
 
     def set_payload(self, payload):
-        self.put_attachment(payload, self.get_attachment_name())
+        # According to @esoergel this code is slated for removal in the near
+        # future, so I didn't think it was worth it to try to pass the domain
+        # in here.
+        domain = "<unknown>"
+        self.put_attachment(payload, self.get_attachment_name(), domain=domain)
```

Comment:
what would be the difference in passing the `domain` though?

Reply:
It's required by `blobdb.put(...)`, but `SavedBasicExport` doesn't have a `domain` attribute.

Reply:
Interesting. I never thought there would be exports not bound to a domain.